Configurations/descrip.mms.tmpl

```diff
@@ -47,13 +47,13 @@
       map { (my $x = $_) =~ s/\.a$//; $x }
       @{$unified_info{libraries}};
   our @shlibs =
-      map { $unified_info{sharednames}->{$_} || () }
+      map { $unified_info{sharednames}->{$_}.$shlibvariant || () }
```

Comment:
Sorry for the stupid question, but where does `$shlibvariant` come from? I see

`sub shlibvariant        { $target{shlib_variant} || "" } `

but presumably there's an `eval` somewhere that creates the scalar?

Reply:
Perfectly valid question!  I had forgotten to define that variable.  Fixed

Reply:
Ah, the perils of legacy Perl code without `use strict`!